src/components/RightSidebar/Participants/Participant.vue

```diff
@@ -4,154 +4,128 @@
 -->
 
 <template>
-	<component :is="tag"
+	<NcListItem :name="computedName"
```

Comment:
```suggestion
	<NcListItem 
```

as you are using name slot

Reply:
Prop is required, so test will scream about it